src/common/zscore/cgmzscore/src/main.py

```diff
@@ -0,0 +1,103 @@
+from constant import REPO_DIR, WHO_TABLES
+from table import Table
+from extract_value import LMS_BOX_COX
+from calculate import Zscore
+from decimal import Decimal as D
+import json
+
+
+TABLE_REPO = REPO_DIR / 'tables'
+GROWTH_CHART = {}
+
+# This set of code should call as soon as we import our library
+# Where we should put this code??
+for t in WHO_TABLES:
+    table_name, underscore, z_score_part = t.split('.')[0].rpartition('_')
```

Comment:
This is unused so maybe we don't need it as a variable with a name?

```suggestion
    table_name, _, _ = t.split('.')[0].rpartition('_')
```

Or we indicate that this is unused like this: 

```suggestion
    table_name, _underscore, _z_score_part = t.split('.')[0].rpartition('_')
```